omnibus/config/software/datadog-agent.rb

```diff
@@ -62,9 +61,22 @@
   env = with_standard_compiler_flags(with_embedded_path(env))
 
   # Use msgo toolchain when fips mode is enabled
-  if fips_mode? && !windows_target?
-    env["GOROOT"] = msgoroot
-    env["PATH"] = "#{msgoroot}/bin:#{env['PATH']}"
+  if fips_mode?
+    if windows_target?
+      msgoroot = ENV['MSGO_ROOT']
+      if msgoroot.nil? || msgoroot.empty?
+        raise "MSGO_ROOT not set"
+      end
+      if !File.exist?("#{msgoroot}\\bin\\go.exe")
+        raise "msgo go.exe not found at #{msgoroot}\\bin\\go.exe"
+      end
+      env["GOROOT"] = msgoroot
+      env["PATH"] = "#{msgoroot}\\bin;#{env['PATH']}"
```

Comment:
nit: could these 2 lines be shared for windows/linux at the expense of using forward slashes?

Reply:
the path delimiter is different, too, semicolon vs colon

Reply:
Oh right, I keep forgetting about this one :facepalm: